pymeasure/instruments/srs/sr570.py

```diff
@@ -0,0 +1,214 @@
+        amplifier, in the [-5V,+5V] limits. 
+        The values are up to 1 mV precision level.""", 
+        validators=truncated_range,
+        values=BIAS_LIMITS,
+        set_process = lambda v: int(1000*v))
+    
+    offset_current = Instrument.setting(
+        "BSLV %f", 
+        """ A floating point value in A that sets the absolute value 
+        of the offset current of the amplifier, in the [1pA,5mA] limits. 
+        The offset current takes discrete values in a 1-2-5 sequence. 
+        Values are truncated to the closest allowed value if not exact. """, 
+        validators=truncated_discrete_set,
+        values=OFFSET_CURRENTS,
+        map_values = True)
+    
+    offset_current_sign = Instrument.setting(
+        "IOSN %d", 
+        """ An string that sets the offset current sign. 
+        Allowed values are: 'positive' and 'negative'. """, 
+        validators=strict_discrete_set,
+        values={'positive': 1, 'negative': 0},
+        map_values=True)
+    
+    gain_mode = Instrument.setting(
+        "GNMD %d", 
+        """ A string that sets the gain mode.
+        Allowed values are: {}""".format(GAIN_MODES), 
+        validators=truncated_discrete_set,
+        values = GAIN_MODES,
+        map_values=True)
+    
+    invert_signal_sign = Instrument.setting(
+        "INVT %d", 
+        """ An boolean sets the signal invert sense.
+        Allowed values are: True (inverted) and False (not inverted). """, 
+        validators=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True)
+    
+    bias_enabled = Instrument.setting(
+        "BSON %d", 
+        """ Boolean that turns the bias on or off.
+        Allowed values are: True (bias on) and False (bias off)""", 
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True)
+    
+    offset_current_enabled = Instrument.setting(
+        "IOON %d", 
+        """ Boolean that turns the offset current on or off.
+        Allowed values are: True (current on) and False (current off).""", 
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True)
+    
+    front_blanked = Instrument.setting(
+        "BLNK %d", 
+        """ Boolean that blanks(True) or un-blanks (False) the front panel""", 
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True)
+    
+    signal_inverted = Instrument.setting(
+        "INVT %d", 
+        """ Boolean that inverts the signal if True""", 
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True)
+    
+    ####################
+    # Methods        #
+    ####################
+    
+    def enable_bias(self):
+        """Turns the bias voltage on"""
+        self.write("BSON 1")
+    
+    def disable_bias(self):
+        """Turns the bias voltage off"""
+        self.write("BSON 0")
```

Comment:
As you have implemented the `bias_enabled` property, maybe you could change `self.write("BSON 1")` to `self.bias_enabled = True` (same holds for the other wrappers below)